### DISCUSSION TITLE
Countries section - ability to set default view to list, not map

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1376] [Anything Else]
Currently, the countries section shows a map by default and for seeing the list of countries, I have to click "Details".

I'd like to change a setting to be shown the countries list by default in that section. Then, I would not have to click on "Details" every time I want to see the numbers from top countries.

(feedback via email)

**Comment by Vigasaurus** [comment]
I really like this, and would want it personally too. I'm happy to pick it up and PR - what are thoughts on where this setting should go/if it should be generalized?

i.e should it be in settings, or as a local storage synced minimal switch in the report itself

and for it to be titled "Prefer table views over graphics" to allow for same setting to reused in future reports, or have it phrased to be map-specific?

**Reply by metmarkosaric** [reply]
We can just continue the current way of doing things rather than introduce settings that few people would change from the default.

We're working on introducing regions/states and cities too. So we could have "countries", "regions" and "cities" like we have "source", "medium", "campaign" in Top Sources. 

And then save whatever is selected last for the next visit too.

**Reply by Vigasaurus** [reply]
So just to be clear, you mean something like the tabs for selecting regions/cities etc (eventually), but as far as selecting the list view/map view - were you thinking something along these lines, or instead, where would that toggle go?

![image](https://user-images.githubusercontent.com/19434157/136408557-a06531ff-c434-4866-bc5a-2a4029ece735.png)

**Reply by metmarkosaric** [reply]
like that but without that toggle. we can list "map" like we have "all" in Top Sources so you click "map" for the map view while all other tabs give you the list view

**Reply by Vigasaurus** [reply]
Gotcha - that makes sense.

**Comment by metmarkosaric** [comment]
This is now live!
